tools/cockpit.spec

```diff
@@ -516,6 +521,10 @@ getent group cockpit-wsinstance >/dev/null || groupadd -r cockpit-wsinstance
 getent passwd cockpit-wsinstance >/dev/null || useradd -r -g cockpit-wsinstance -d /nonexisting -s /sbin/nologin -c "User for cockpit-ws instances" cockpit-wsinstance
 
 %post ws
+%if 0%{?suse_version}
+%set_permissions %{_libexecdir}/cockpit-session
```

Comment:
What does that do in addition to

    %attr(4750, root, cockpit-wsinstance) %{_libexecdir}/cockpit-session

?

Reply:
it's the SUSE runtime permission handling. we have permission profiles so can change setuid bits according to run time config. https://github.com/openSUSE/permissions/